### ISSUE TITLE
Why Slow

### ISSUE THREAD

**Issue by Muhammed0907** [ximinng/PyTorch-SVGRender#22]
Hello, greate project, but why it is so slow, i wanna do image to svg, but it so slow it is doing 2000iter. how can i make it faster, i also use cuda, still slow

**Comment by ximinng** [comment]
As you can see, it takes 2000 iters, and you can try to reduce the number of iterations and increase the learning rate
